handel/src/aggregation.rs

```diff
@@ -54,25 +58,26 @@ where
     /// Our contribution
     contribution: P::Contribution,
 
-    /// Sink used to relay messages
-    sender: LevelUpdateSender<N>,
+    /// Gateway to the network to send updates and ban nodes.
+    network: NetworkHelper<N>,
 
-    /// Interval for starting the next level regardless of previous levels completion
+    /// Timeout for starting the next level regardless of previous level's completion.
```

Comment:
```suggestion
    /// Interval for starting the next level regardless of previous level's completion.
```
Or also change the comment for `periodic_update_interval` below.